src/topology/math.py

```diff
@@ -0,0 +1,58 @@
+from enum import Enum
+from math import atan2, cos, sin, degrees, radians, sqrt
+
+from data.measurement_data import Coordinates
+
+import geopy.distance as geo
+import pyproj as proj
+
+from initial_values import InitialValues
+
+
+class Move_direction(Enum):
+    North = 0
+    East = 90
+    South = 180
+    West = 270
+
+def coordinates_distance(first: Coordinates, second: Coordinates) -> float:
+    return geo.distance(first.as_tuple(), second.as_tuple()).meters
+
+def move_coordinate_bearing(source: Coordinates, distance: float, bearing: float) -> Coordinates:
+    point = geo.distance(meters=distance).destination(source.as_tuple(), bearing)
+    return Coordinates(latitude=point.latitude, longitude=point.longitude)
+
+def move_coordinate(source: Coordinates, distance: float, direction: Move_direction) -> Coordinates:
+    return move_coordinate_bearing(source, distance, direction.value)
+
+def move_coordinate_raw(lat: float, lon: float, distance: float, direction: Move_direction) -> (float, float):
```

Comment:
In return type hints use Tuple[], applicable everywhere I posted comment about redundant parenthesis